### DISCUSSION TITLE
Avalonia.Remote.Protocol.dll what is it for and how Avalonia internals use it?

### DISCUSSION THREAD

**Discussion by Zilkode** [AvaloniaUI/Avalonia#7683] [Q&A] [answered]
As far as I know Avalonia is about UI, why there is remoting inside it?
Is there any internal calls to this assembly that i am unaware of?

Thanks.

**Comment by jp2masa** [comment]
I think it's only used by the previewer.

**Comment by maxkatz6** [comment, marked as answer]
Currently it's used for Designer (Avalonia.DesignerSupport project).
Also, it might be used for Dev Tools later, if we decide to move the out of process.
You always can check an open source.
